test/src/main/java/org/apache/accumulo/test/BulkImportMonitoringIT.java

```diff
@@ -65,15 +67,21 @@ protected void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSit
   public void test() throws Exception {
     getCluster().getClusterControl().start(ServerType.MONITOR);
     try (AccumuloClient c = Accumulo.newClient().from(getClientProperties()).build()) {
+
+      // creating table name
       final String tableName = getUniqueNames(1)[0];
-      c.tableOperations().create(tableName);
-      c.tableOperations().setProperty(tableName, Property.TABLE_MAJC_RATIO.getKey(), "1");
-      // splits to slow down bulk import
+      // creating splits
       SortedSet<Text> splits = new TreeSet<>();
       for (int i = 1; i < 0xf; i++) {
         splits.add(new Text(Integer.toHexString(i)));
       }
```

Comment:
A lot of these loops could be converted to a nice one-(ish)-liner:
```java
TreeSet<Text> splits = IntStream.range(1, 16).mapToObj(Integer::toHexString).map(Text::new).collect(someTreeSetCollector);
```
The main annoying thing is that there isn't a `Collectors.toSortedSet()`, like there is a `Collectors.toSet()`, so you have to write your own collector.

It's probably not worth it.

Reply:
I personally prefer the classic for loop in this situation. I don't think sacrificing the clarity of the traditional loop is worth making it 2 lines shorter.

Reply:
Especially in a test since the traditional loops are simpler for new folks learning the code, either new to Java or the project.

Reply:
@milleruntime I agree with your last two comments. While the stream operations make for tidy code, there are some not so obvious differences between them and for-loops.

Reply:
@milleruntime I would argue that the functional syntax is designed to be mimic human logical thinking more, so it's not necessarily simpler for new folks learning code. I think the traditional loops are more familiar with developers who have been around longer, but they aren't more intuitive to people who are familiar with functional programming, or new to programming in general. The terminal condition in traditional for loops, in particular, is difficult to grok for new programmers, and prone to off-by-one type errors, even for seasoned developers.

However, I agree that it's not worth it here to replace the whole loop. However, it may be worth it to do something like the following, so the terminal condition of the loop is more obvious:

```suggestion
      SortedSet<Text> splits = new TreeSet<>();
      IntStream.range(1, 15).forEach(i -> {
        splits.add(new Text(Integer.toHexString(i)));
      });
```

The reason it's important to make it more clear what the loop condition is, is because many of our tests depend on a specific number of split points, and understanding the test as a whole. In this case, strangely, there are 14 split points, which would mean that there should be tablets for this table in the end. Based on `IntStream.range()` being `inclusive - exclusive`, this means we can write our loops as `IntStream.range(1, NUM_TABLETS)` and it is easy to understand.

Reply:
Regardless, I don't think its fair for @DomGarguilo to include this as part of the code review.

Reply:
> Regardless, I don't think its fair for @DomGarguilo to include this as part of the code review.

Part of this issue seems to be about standardizing the way split points are created in our tests. My suggestion (or some variant) could be included as a part of that, or as a follow on PR, or not at all. It is merely a suggestion, in case they want to consider it. Whether it is included or not, I leave to their discretion.

Reply:
Are you suggesting that all loops be converted or just this specific case which deals with hex values? @ctubbsii

Reply:
Basically, the suggestion was just to opportunistically improve the readability of the split creation loops for the code that you were already modifying, while you were in there. So, not all loops, but also not necessarily just this one... they all probably follow a similar pattern (not always hex, though). But, the value for the change would be minimal, so it's up to you whether you want to apply the suggestion or not, based on your own judgement of whether you think it would be useful.